hunting-buddy.lic

```diff
@@ -76,6 +78,14 @@ class HuntingBuddy
     bput("stow my #{description}", 'You put')
   end
 
+  def check_prehunt_buffs
+    return unless @settings.waggle_sets['prehunt_buffs']
```

Comment:
I did this. One issue is that since buffs will use default if empty arguments, it doesn't natively support the prebuff_buffs without some type of boolean check. Otherwise, it will always prebuff regardless of whether or not you set it up to do so. I added logic to take care of this, let me know if this is easier done in ruby, or cleaner, than I have it now.